test/forge/integration/AccrueInterestIntegrationTest.sol

```diff
@@ -155,8 +161,7 @@ contract AccrueInterestIntegrationTest is BaseTest {
         morpho.supplyCollateral(marketParams, 1, address(this), hex"");
         vm.expectEmit(true, true, true, true, address(morpho));
         emit EventsLib.AccrueInterest(id, params.borrowRate, params.expectedAccruedInterest, params.feeShares);
-        // Accrues interest.
-        morpho.withdrawCollateral(marketParams, 1, address(this), address(this));
+        morpho.accrueInterest(marketParams);
```

Comment:
Same, remove the `collateralToken.setBalance` and `morpho.supplyCollateral` lines above